tools/tentacle_manager/tentacle_util.py

```diff
@@ -207,40 +207,27 @@ def create_localization_from_type(localization, module_type, module_subtype, fil
     # create path from types
     test_folder_if_required = ""
     if tests:
-        test_folder_if_required = "/{0}".format(TENTACLES_TEST_PATH)
+        test_folder_if_required = f"/{TENTACLES_TEST_PATH}"
     if module_subtype:
-        return "{0}{1}/{2}/{3}/{4}".format(localization,
-                                           test_folder_if_required,
-                                           module_type,
-                                           module_subtype,
-                                           file)
+        return f"{localization}{test_folder_if_required}/{module_type}/{module_subtype}/{file}"
     else:
-        return "{0}{1}/{2}/{3}".format(localization,
-                                       test_folder_if_required,
-                                       module_type,
-                                       file)
+        return f"{localization}{test_folder_if_required}/{module_type}/{file}"
 
 
 def create_path_from_type(module_type, module_subtype, target_folder, tests=False):
     # create path from types
     test_folder_if_required = ""
     if tests:
-        test_folder_if_required = "/{0}".format(TENTACLES_TEST_PATH)
+        test_folder_if_required = f"/{TENTACLES_TEST_PATH}"
     if module_subtype:
-        return "{0}{1}/{2}/{3}/{4}".format(TENTACLES_PATH,
-                                           test_folder_if_required,
-                                           TENTACLE_TYPES[module_type],
-                                           TENTACLE_TYPES[module_subtype],
-                                           target_folder)
+        return f"{TENTACLES_PATH}{test_folder_if_required}/{TENTACLE_TYPES[module_type]}/" \
```

Comment:
I think that a [_Multi-line string syntax_](https://www.smallsurething.com/multi-line-strings-in-python/) will be better here.